english/call-a-doctor/doctor.md

```diff
@@ -5,7 +5,7 @@ color, not like the institutionalized white you remember from movies.
 
 [The doctor](https://www.youtube.com/watch?v=WsGDvMqcP94) enters.
 
-["Let me outta here, Doc!!!"](let-me-out/let-me-out.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.